client/verta/verta/endpoint/build/_build.py

```diff
@@ -102,3 +102,30 @@ def get_scan(self) -> _build_scan.BuildScan:
 
         """
         return _build_scan.BuildScan._get(self._conn, self.id)
+
+    def start_scan(self, external: bool = False) -> _build_scan.BuildScan:
+        """Start a new scan for this build. Internal scans are not yet supported.
+        Use ``external=True`` parameter.
+
+        This function only awaits a success response for the request, not any result
+        from the scan itself. Use
+        :meth:`Build.get_scan() <verta.endpoint.build.Build.get_scan()>` to view
+        progress and results of the scan.
+
+        Parameters
+        ----------
+        external : bool, default False
+            True if using an external scan provider.
+
+        Returns
+        -------
+        :class:`~verta.endpoint.build.BuildScan`
+            Build scan.
+
+        """
+        if not external:
+            raise NotImplementedError(
+                "internal scans are not yet supported.  Please use `external=True`"
+                " parameter."
```

Comment:
Don't use trailing periods in exception messages.

```suggestion
                "internal scans are not yet supported; please use `external=True`"
                " parameter"
```

Reply:
Fixed